libopenage/engine.cpp

```diff
@@ -548,4 +542,12 @@ void Engine::move_phys_camera(float x, float y, float amount) {
 	this->engine_coord_data->camgame_phys += cam_delta.to_phys3();
 }
 
+void Engine::start_game(std::unique_ptr<GameMain> game) {
+	// TODO: maybe implement a proper 1-to-1 connection
+	assert(game);
```

Comment:
moc doesn't throw and it doesn't copy-paste user-written code (so it doesn't throw).

Explicit is better than implicit, but implicit is better then none. And you get none if the syntax is more complex than an assert. It's like that for a reason. You normally move and change dozens of asserts while developing.